utils/k8s/openebscrd.go

```diff
@@ -56,6 +56,13 @@ func (k8s K8S) DeleteCStorPool(cStorPoolName string, opts *meta_v1.DeleteOptions
 	return cStorePoolClient.Delete(cStorPoolName, opts)
 }
 
+// CreateStoragePool takes the representation of a StoragePool and creates it.
+// :return: *openebs_v1.StoragePool: Pointer to StoragePool object.
```

Comment:
Can we please remove this comment at line 60.
We do not require this in my opinion. ( And where ever it is in current code, we will be removing it)